src/libraries/Native/native-binplace.proj

```diff
@@ -20,8 +20,21 @@
       <BinPlaceItem Include="$(NativeBinDir)*.dbg" />
       <BinPlaceItem Include="$(NativeBinDir)*.dylib" />
       <BinPlaceItem Include="$(NativeBinDir)*.dwarf" />
+      <BinMobileNative Condition="'$(BinPlaceTestRuntimePack)' == 'true'" Include="@(BinPlaceItem)" />
```

Comment:
the condition on this node can't be true if the condition on the parent ItemGroup is true